invokeai/app/invocations/latent.py

```diff
@@ -360,11 +388,21 @@ def _lora_loader():
                     exit_stack=exit_stack,
                 )
 
+                num_inference_steps, timesteps = self.init_scheduler(
+                    scheduler,
+                    device=unet.device,
+                    steps=self.steps,
+                    denoising_start=0.0,
+                    denoising_end=self.denoising_end,
+                )
```

Comment:
It seems like `t2l` and `l2l` can almost be merged? If I understand correctly, the differences are:
- `t2l` starts with full-noise while `l2l` starts with partial-noise
- `t2l` always denoises from 0, while `l2l` denoises from user-defined value

What else is different? Do we even need two nodes?

Reply:
Thanks for bringing this up!

You are correct, these are not different operations.

It gets `latents` as input, and sometimes those latents will be right straight out of the Gaussian noise generator, and sometimes they'll be from something else. So there isn't need for a separate `noise` input (unless that's a Generator or something).